spec/boac/boac_sis_data_spec.rb

```diff
@@ -564,8 +578,10 @@
           row = [student.uid, student_page_sis_data[:name], student_page_sis_data[:preferred_name], student_page_sis_data[:email],
                  student_page_sis_data[:email_alternate], student_page_sis_data[:phone], student_page_sis_data[:cumulative_units],
                  student_page_sis_data[:cumulative_gpa], student_page_sis_data[:level], student_page_sis_data[:transfer],
-                 student_page_sis_data[:colleges], student_page_sis_data[:majors], student_page_sis_data[:colleges_discontinued],
-                 student_page_sis_data[:majors_discontinued], student_page_sis_data[:minors], student_page_sis_data[:terms_in_attendance],
+                 student_page_sis_data[:colleges], student_page_sis_data[:majors],
+                 student_page_sis_data[:colleges_discontinued], student_page_sis_data[:majors_discontinued],
+                 student_page_sis_data[:minors], student_page_sis_data[:minors_discontinued],
```

Comment:
Hey @paulkerschen - you should update the CSV column headers that are set at the beginning of the script if you're inserting a column. Line 15 I think.

Reply:
Thanks @pfarestveit - done.